integration_test/gce/gce_testing.go

```diff
@@ -359,14 +353,6 @@ func IsWindows(platform string) bool {
 	return strings.HasPrefix(platform, "windows-") || strings.HasPrefix(platform, "sql-")
 }
 
-// PlatformKind returns "linux" or "windows" based on the given platform.
```

Comment:
This definitely looks like a merge issue — it undoes charges from #719.

Reply:
bad merge